idl/Microsoft.UI.Xaml.idl

```diff
@@ -158,10 +158,10 @@ namespace MU_XC_NAMESPACE
 {
     [WUXC_VERSION_PREVIEW]
     [webhosthidden]
-    enum StylesVersion
+    enum ControlsResourceVersion
```

Comment:
This aligns with XamlControlsResources (note the plural "Resources"), so should be ControlsResourcesVersion

Reply:
make sense